server/website/website/tasks/async_tasks.py

```diff
@@ -932,12 +932,16 @@ def map_workload(map_workload_input):
             # and then predict the performance of each metric for each of
             # the knob configurations attempted so far by the target.
             y_col = y_col.reshape(-1, 1)
-            model = GPRNP(length_scale=params['GPR_LENGTH_SCALE'],
-                          magnitude=params['GPR_MAGNITUDE'],
-                          max_train_size=params['GPR_MAX_TRAIN_SIZE'],
-                          batch_size=params['GPR_BATCH_SIZE'])
-            model.fit(X_scaled, y_col, ridge=params['GPR_RIDGE'])
-            predictions[:, j] = model.predict(X_target).ypreds.ravel()
+            model_kwargs = {'lengthscales': params['GPR_LENGTH_SCALE'],
```

Comment:
Do we need a configuration to control which algorithm to use? I prefer that we only use the GPFlow version if it is correct. Then we should consider removing the GPRGD as well.